helm/values.yaml

```diff
@@ -47,55 +51,58 @@ validators:
   #   password: notarypass
   #   # or
   #   secret_name: notarysecret  # define a k8s secret with `username` and `password` fields
-- name: docker_essentials  # public notary form dockerhub
+- name: dockerhub_essentials   # pre-configured validator for public notary from dockerhub
   type: notaryv1
   host: notary.docker.io
   pub_keys:
-  # this is the public key for all official dockerhub images (e.g. redis)
-  - name: docker_official
+  - name: docker_official   # public key for official docker images (e.g. redis)
     key: |
       -----BEGIN PUBLIC KEY-----
       MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEOXYta5TgdCwXTCnLU09W5T4M4r9f
       QQrqJuADP6U7g5r9ICgPSmZuRHP/1AYUfOQW3baveKsT969EfELKj1lfCA==
       -----END PUBLIC KEY-----
-  # this is the public key to the `securesystemsengineering` repo on dockerhub
-  - name: sse_official
+  - name: sse_official      # public key for connaisseur images
     key: |
       -----BEGIN PUBLIC KEY-----
       MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEsx28WV7BsQfnHF1kZmpdCTTLJaWe
       d0CA+JOi8H4REuBaWSZ5zPDe468WuOJ6f71E7WFg3CVEVYHuoZt2UYbN/Q==
       -----END PUBLIC KEY-----
-- name: cosign_demo  # for verifying cosign signatures
+- name: cosign_essentials   # pre-configured validator for public cosign signatures
   type: cosign
   pub_keys:
-  - name: sse
+  - name: sse_official      # public key for connaisseur images (currently only testimages)
```

Comment:
What test images?

Also, time dependent statements (like "currently") are prone to become outdated.

Reply:
we maintain a few signed testimages on dockerhub that we for example use for integration tests. We could use the official key also to sign connaisseur in the future, however that is not being done for now, as everything is experimental

Reply:
Why is testing stuff in the configuration that we ship by default?

Reply:
we can use the shipped cosign key to also sign connaisseur itself in the future. otherwise, it may make getting started easier. I also added the testimages in the image policy below. That is sort of nice to demo, but might be conflicting with the default configuration. Do you think I should remove it?

Reply:
I think @Starkteetje can weigh in here.

Reply:
Afaik, and as documented in the [RELEASING.md](https://github.com/sse-secure-systems/connaisseur/blob/master/RELEASING.md) we do sign the production Connaisseur images. Then adding our pubkey as config with the image policy prescribing said key for our images is worthwhile I'd say. If it were only for testing, I'd remove it instead, but as it actually has sec benefits. I'd keep it as is